test/e2e/framework/perf_util.go

```diff
@@ -51,6 +51,26 @@ func ApiCallToPerfData(apicalls *APIResponsiveness) *perftype.PerfData {
 	return perfData
 }
 
+// PodStartupLatencyToPerfData transforms PodStartupLatency to PerfData.
+func PodStartupLatencyToPerfData(latency *PodStartupLatency) *perftype.PerfData {
+	perfData := &perftype.PerfData{Version: currentApiCallMetricsVersion}
+	item := perftype.DataItem{
+		Data: map[string]float64{
+			"Perc50":  float64(latency.Latency.Perc50) / 1000000, // us -> ms
+			"Perc90":  float64(latency.Latency.Perc90) / 1000000,
+			"Perc99":  float64(latency.Latency.Perc99) / 1000000,
+			"Perc100": float64(latency.Latency.Perc100) / 1000000,
+		},
+		Unit: "ms",
+		Labels: map[string]string{
+			"Verb":     "POD_STARTUP",
```

Comment:
I don't like this. I don't think this should be Verb, this should be operation or sth like that.